### PR TITLE
Quick fix  for the translate composer + chrome 62.

### PR DESCRIPTION
**Pull request by thibaultzanini**
Need to be removed when the EventManager will handle the parameter options of the addEventListener function.

### CODE REVIEW

**Comment by thibaultzanini** [comment]
This PR disable the HttpService specs, it seems they are broken.
https://github.com/montagejs/montage/issues/1898

**Comment by hthetiot** [comment]
Test ok on ios and android:
- http://montagejs.github.io/popcorn/

Still we might need to fix that on desktop.

screen shot 2017-11-16 at 2 08 08 pm

File: composer/press-composer.info/sample/ui/draggable-square.reel/draggable-square.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
+.DraggableSquare {
+    touch-action: none;
```
[the remark this answers is not in the document]
Comment (thibaultzanini) [reply composer/press-composer.info/sample/ui/draggable-square.reel/draggable-square.css]: I did not find any regression so far

**Comment by hthetiot** [comment]
https://github.com/montagejs/montage/pull/1866

**Pull request merged**